common/changes/@typespec/eslint-config-typespec/versioning-removeDeprecated_2023-03-17-18-06.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
None of the rest of these change entries should be required

Reply:
other than the versioning library

Reply:
The unnecessary ones are removed.